modules/citrus-http/src/main/java/com/consol/citrus/http/servlet/CachingHttpServletRequestWrapper.java

```diff
@@ -55,7 +55,7 @@ public CachingHttpServletRequestWrapper(HttpServletRequest request) {
 
         Map<String, String[]> params = new HashMap<>();
         if (RequestMethod.POST.name().equals(getMethod()) || RequestMethod.PUT.name().equals(getMethod())) {
-            if ("application/x-www-form-urlencoded".equals(getContentType())) {
+            if (getContentType() != null && getContentType().contains("application/x-www-form-urlencoded")) {
```

Comment:
`.contains()` is unfortunately insufficient for the check.
This would also allow `foo-application/x-www-form-urlencoded-bar`.
One could think about using `getContentType().startsWith("application/x-www-form-urlencoded")` here.

Reply:
Thank you for your response! 
Now I see that it is not that simple. I have added an issue. Hope it would be fixed. 
I'm afraid I don't have enough competence to correct you about RFC2045 specification, but my "system under test" (which is springboot based application) sends exactly such Content-Type header in requests.